modules/geo-layers/src/h3-layers/h3-hexagon-layer.ts

```diff
@@ -99,6 +100,13 @@ function mergeTriggers(getHexagon, coverage) {
   return trigger;
 }
 
+function checkH3Lib() {
```

Comment:
Can you move this to `bundle.ts`? We try not to include long assert messages in the library to control the bundle size.